clean_bistreams.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
 
+CURRENT=$(df / | grep / | awk '{ print $5 }' | sed 's/%//g')
+THRESHOLD=95
+
 # Compress bistream files older than 5 minutes
-find /opt/dionaea/var/dionaea/bistreams/* -type f -mmin +5 -exec gzip {} \;
+find /opt/dionaea/var/dionaea/bistreams/* -name '*.gz' -type f -mmin +5 -exec gzip {} \;
```

Comment:
This looks like it would only be attempting to gzip files that are already gzipped (based on the name extension).

Is this intended? I have not yet tested this logic.